crates/air-lib/interpreter-data/src/executed_state.rs

```diff
@@ -39,7 +39,11 @@ pub struct ParResult {
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub enum Sender {
     PeerId(Rc<String>),
-    PeerIdWithCallId { peer_id: Rc<String>, call_id: u32 },
+    PeerIdWithCallId {
+        peer_id: Rc<String>,
+        call_id: u32,
+        argument_hash: Rc<str>,
```

Comment:
Why do we need an argument hash here?

Reply:
As discussed: it was intended to save from undefined behavior in variable changing values in `par` branches, but it was suggested to not to do it to reduce data size, moreover, this undefined behavior prevents other peers to verify hashes, and requires other solution.